atlasdb-commons/src/main/java/com/palantir/common/base/Throwables.java

```diff
@@ -82,6 +83,13 @@ public static RuntimeException rewrapAndThrowUncheckedException(String newMessag
      * If Throwable is a RuntimeException or Error, rethrow it. If not, throw a
      * new PalantirRuntimeException(ex)
      */
+    public static RuntimeException unwrapAndThrowUncheckedException(Throwable ex) {
+        if (isInstance(ex, ExecutionException.class) || isInstance(ex, InvocationTargetException.class)) {
+            throwUncheckedException(ex.getCause());
```

Comment:
Typical convention is to throw anything this method returns, to make it obvious that this line throws.